manifests/cluster/patch/pg-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+- op: add
+  path: "/data"
+  value:
+    default_cpu_request: 500m
+    default_memory_request: 250Mi
+    api_port: "8080"
```

Comment:
```suggestion

```

Reply:
all the items under this line can be removed. After the change, you can run `kustomize build ./ -o deployment.yaml` to check if the data in the cfgMap are matching our expectations.

Reply:
change pg-configmap.yaml to:

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: postgres-operator
data:
  default_cpu_request: 500m
  default_memory_request: 250Mi
```

and use patchesStrategicMerge:

```yaml
# If you have to override the image path, uncomment this patch and also change the image paths in the patch/patches-images.yaml file.
# Then rebuild.
patchesStrategicMerge:
  - patch/pg-configmap.yaml
# - patch/images.yaml
```

cc @Ghostbaby